src/app/wallet/wallet/send/send.component.ts

```diff
@@ -120,6 +121,11 @@ export class SendComponent {
       this.send.validAmount = false;
       return;
     }
+
+    if (this.send.amount === 1e-8) {
```

Comment:
@vikas-cis  can we set amount `0` if the amount equals to `1e-8` ?

Reply:
@anandsinghparihar its not `0` it is 1 SAT I guess which is a valid amount. So I think we don't need to show `0`